support/routes/v2/isobot/index.ts

```diff
@@ -0,0 +1,44 @@
+import express, { RequestHandler } from "express"
+
+import { Whitelist } from "@database/models"
+import logger from "@root/logger/logger"
+import WhitelistService from "@root/services/identity/WhitelistService"
+
+import BotService, { SlackPayload } from "./ops/botService"
+
+export const isobotRouter = express.Router()
+const botService = new BotService(
+  new WhitelistService({ repository: Whitelist })
+)
+
+const handleWhitelistEmails: RequestHandler<
+  {},
+  { message: string },
+  SlackPayload,
+  {},
+  {}
+> = async (req, res) => {
+  console.log(req.headers)
+  const slackTimestamp = req.headers["x-slack-request-timestamp"] as string
+  const slackSig = req.headers["x-slack-signature"] as string
+
+  if (!slackTimestamp || !slackSig)
+    return res.send({ message: "Missing header/signature" })
+
+  const isVerifiedMessage = botService.verifySignature(
```

Comment:
nit: i think we should also limit this method to 
1. isomer prod ops
2. isomer engineers 

so that the potential for accidents are minimised. additionally, we might benefit from writing any such changes into the `isomer-alarm` channel so we get clarity on who does what

Reply:
resolving, done in #1391